quarkus/extensions/src/main/java/org/keycloak/provider/quarkus/JsonConfigProviderFactory.java

```diff
@@ -17,6 +17,25 @@
 
 package org.keycloak.provider.quarkus;
 
+import java.util.Optional;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.keycloak.Config;
+import org.keycloak.common.util.SystemEnvProperties;
+import org.keycloak.services.util.JsonConfigProvider;
+
 public class JsonConfigProviderFactory extends org.keycloak.services.util.JsonConfigProviderFactory {
 
+    protected Optional<Config.ConfigProvider> createJsonProvider(JsonNode node) {
```

Comment:
Why is this being added?

Reply:
This is related to what we discussed in regards to resolving env vars as a fallback.

Reply:
Not sure what you mean, but the config provider Dimitry is working on should automatically support both keycloak.properties, system properties and environment variables.

Reply:
Yeah, which we don't have at the moment so we can at least check the behavior regardless of how it is implemented. Now that we have the PR from Dimitry I can remove this.